icons/mixpanel.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Mixpanel</title><path d="M23.684 12.717a.316.316 0 0 1-.315-.32V8.32c0-.186.131-.32.315-.32.185 0 .316.15.316.32v4.077c0 .17-.131.32-.316.32m-2.172-2.622c-.463 0-.778.34-.852.774h1.668c-.053-.433-.353-.774-.816-.774Zm1.41 1.074c0 .114-.053.227-.242.227h-2.02c.037.47.352.83.906.83.356 0 .557-.133.782-.34.054-.056.127-.113.205-.113.148 0 .258.114.258.263 0 .077-.036.134-.073.191-.263.3-.726.51-1.21.51-1.02 0-1.536-.717-1.536-1.584 0-.908.647-1.585 1.52-1.585.853 0 1.41.714 1.41 1.601zm-4.75-1.6c.816 0 1.426.64 1.426 1.45v1.378a.316.316 0 1 1-.63 0v-1.455a.759.759 0 0 0-.78-.77.759.759 0 0 0-.778.77v1.455a.316.316 0 1 1-.631 0V11.02c-.02-.81.573-1.45 1.393-1.45m-3.471.563c-.541 0-.93.474-.93 1.037 0 .604.315 1.038.93 1.038.557 0 .926-.49.926-1.038-.016-.66-.39-1.037-.926-1.037zm1.557 2.265c0 .187-.131.32-.315.32a.316.316 0 0 1-.316-.32v-.15a1.2 1.2 0 0 1-.984.507c-.946 0-1.504-.718-1.504-1.585s.615-1.584 1.578-1.584c.984 0 1.562.66 1.562 1.584v1.228zm-4.918-2.281c-.557 0-.91.49-.91 1.053 0 .547.373 1.058.91 1.058.558 0 .91-.417.91-1.058 0-.64-.352-1.053-.91-1.053zm.131 2.638a1.17 1.17 0 0 1-1.04-.604v1.548c0 .186-.132.32-.316.32a.316.316 0 0 1-.316-.32V9.925c0-.19.11-.32.295-.32.189 0 .316.15.316.34v.243a1.237 1.237 0 0 1 1.061-.603c.779 0 1.41.66 1.41 1.584 0 .944-.631 1.585-1.41 1.585zM7.98 10.715l.815-1.017A.3.3 0 0 1 9 9.605c.184 0 .3.113.3.284 0 .056-.021.13-.075.186l-.893 1.078.91 1.094c.057.053.074.11.074.187a.29.29 0 0 1-.28.283.263.263 0 0 1-.22-.113L8 11.566l-.82 1.038a.253.253 0 0 1-.221.113.29.29 0 0 1-.279-.283.24.24 0 0 1 .074-.187l.91-1.094-.89-1.078a.26.26 0 0 1-.073-.186c0-.17.11-.284.295-.284.094 0 .147.036.205.093l.779 1.017m-2.115 2.002a.313.313 0 0 1-.316-.32V9.925c0-.19.127-.32.316-.32.184 0 .315.15.315.32v2.472c0 .17-.13.32-.315.32zm0-4.356c.205 0 .369.17.369.377a.373.373 0 0 1-.37.376.377.377 0 0 1 0-.753zm-1.226 4.356a.316.316 0 0 1-.315-.32V11c0-.47-.226-.847-.726-.847s-.815.397-.815.847v1.378c0 .19-.131.32-.316.32a.316.316 0 0 1-.315-.32v-1.394c0-.474-.242-.851-.742-.851-.52 0-.779.397-.779.85v1.375c0 .19-.131.324-.315.324A.316.316 0 0 1 0 12.357V9.925c0-.19.127-.34.316-.34.22 0 .315.15.315.34v.227c.185-.36.553-.584.963-.584.484 0 .853.244 1.041.62.258-.413.648-.62 1.131-.62.816 0 1.226.584 1.226 1.395v1.434a.349.349 0 0 1-.353.32m3.64 2.246a.34.34 0 0 1-.336.34.337.337 0 0 1-.332-.34.334.334 0 1 1 .668 0m-2.693 0a.752.752 0 0 1-.742.753.753.753 0 0 1-.746-.753.75.75 0 0 1 .746-.754.75.75 0 0 1 .742.754m-3.471-.02c0 .583-.463 1.057-1.041 1.057a1.047 1.047 0 0 1-1.037-1.058c0-.583.463-1.058 1.037-1.058.578 0 1.04.475 1.04 1.058"/></svg>
```

Comment:
I think the icon should be just the three circles without the wordmark. You can use the official `Brand Mark-Purple.svg` from the [press kit](https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9) whose link you can find near the bottom of https://mixpanel.com/news/. From looking at the press kit, I even think that the official icon isn't just three circles but a square with rounded corners and three circles inside.